symengine/lib/symengine_wrapper.pyx

```diff
@@ -37,6 +37,12 @@ cdef c2py(RCP[const symengine.Basic] o):
     elif (symengine.is_a_Pow(deref(o))):
         r = Basic.__new__(Pow)
     elif (symengine.is_a_Integer(deref(o))):
+        if (deref(symengine.rcp_static_cast_Integer(o)).is_zero()):
+            return S.Zero
+        elif (deref(symengine.rcp_static_cast_Integer(o)).is_one()):
+            return S.One
+        elif (deref(symengine.rcp_static_cast_Integer(o)).is_minus_one()):
+            return S.NegativeOne
```

Comment:
True. So wouldn't a class be better for that? One that is not used in `c2py` but can be used to create an instance of `Zero`. As `Zero()` is also used in `SymPy` at some places?